OpenAPI/decentralized-iso20022-v1.yaml

```diff
@@ -1,152 +1,395 @@
 openapi: 3.0.3
 info:
   title: Decentralized ISO 20022 Communication Service
-  description: This is the API specificaiton for the APIs in the decentralized ISO 20022 communication service.
+  description: This is the API specificaiton for the external API in the decentralized ISO 20022 communication service.
```

Comment:
Why was this changed to external API?

Reply:
This was initially changed when I was going to do a separate document for external and internal but realized that the API has the same structure. Will change this back. 👍